stream-chat-android-client/src/main/java/io/getstream/chat/android/client/models/Device.kt

```diff
@@ -2,18 +2,13 @@ package io.getstream.chat.android.client.models
 
 /**
  * Device's information needed to register push notifications
```

Comment:
Let's add missing punctuation to this KDoc

Reply:
:+1: